app.json

```diff
@@ -13,7 +13,8 @@
       "description": "development environment",
       "value": "development"
     },
-    "NPM_CONFIG_PRODUCTION": {
+  },
```

Comment:
Look and see if this is an extra bracket or if its needed. heroku deployment is currently not working for this PR.